lang/cs/Org.Apache.REEF.IMRU/API/IMRUGenericTypePipelineDataConverterConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using Org.Apache.REEF.Network.Group.Pipelining;
+using Org.Apache.REEF.Tang.Formats;
+using Org.Apache.REEF.Tang.Util;
+
+namespace Org.Apache.REEF.IMRU.API
+{
+    /// <summary>
+    /// A configuration module for The PipelineDataConverter used for chunking 
+    /// and dechunking TMapInput and TMapOutput
+    /// </summary>
+    /// <typeparam name="T">Generic type</typeparam>
+    public sealed class IMRUGenericTypePipelineDataConverterConfiguration<T> : ConfigurationModuleBuilder
```

Comment:
Renames and pushed

On Thu, Aug 6, 2015 at 2:35 PM, Markus Weimer <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.IMRU/API/IMRUGenericTypePipelineDataConverterConfiguration.cs
> <https://github.com/apache/incubator-reef/pull/334#discussion_r36468946>:
>
> > + * specific language governing permissions and limitations
> > + * under the License.
> > + */
> > +
> > +using Org.Apache.REEF.Network.Group.Pipelining;
> > +using Org.Apache.REEF.Tang.Formats;
> > +using Org.Apache.REEF.Tang.Util;
> > +
> > +namespace Org.Apache.REEF.IMRU.API
> > +{
> > +    /// <summary>
> > +    /// A configuration module for The PipelineDataConverter used for chunking
> > +    /// and dechunking TMapInput and TMapOutput
> > +    /// </summary>
> > +    /// <typeparam name="T">Generic type</typeparam>
> > +    public sealed class IMRUGenericTypePipelineDataConverterConfiguration<T> : ConfigurationModuleBuilder
>
> Yes, but why such a long name? Maybe
> IMRUPipelineDataConverterConfiguration?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/334/files#r36468946>.
>